gulpfile.js

```diff
@@ -146,18 +145,17 @@ const underscorePrefix = () => gulpif((f) => f.basename[0] !== '_', rename({ pre
 gulp.task('copy-login-scss', () =>
   gulp
     .src([`${PROJECT_SASS_SRC}/**/*.scss`])
-    .pipe(replace("@import 'uswds'", "@import 'uswds/uswds'"))
     .pipe(replaceUrls())
     .pipe(underscorePrefix())
     .pipe(gulp.dest(SCSS_DEST)),
 );
 
 gulp.task('copy-uswds-scss', () =>
   gulp
-    .src([`${uswds}/scss/**/*.scss`])
+    .src(['node_modules/uswds/dist/scss/**/*.scss'])
     .pipe(replaceUrls())
     .pipe(underscorePrefix())
-    .pipe(gulp.dest(`${SCSS_DEST}/uswds`)),
+    .pipe(gulp.dest(`${SCSS_DEST}/uswds/dist/scss`)),
```

Comment:
This does change the output path for USWDS files from `dist/assets/scss/uwds` to `dist/assets/scss/uswds/dist/scss`. It's not ideal, but avoids some of the "magic" from the renamed import removed above in this file, and it's generally easier to resolve with `includePaths`. I'd hoped this wouldn't be a problem for consumers, but I see now we're referencing some of those paths in the IDP:

https://github.com/18F/identity-idp/blob/c1a0269/config/webpack/environment.js#L39-L41

It wouldn't be much of a problem to update those, and I still don't necessarily consider this a "public" interface, but I'll see if I can adjust this to avoid changing the output directory.

Reply:
Symlink seems like a reasonable option: 4c62728

The other obvious option would be to do the same `cp` of USWDS source files in development that occurs at build-time, which is probably a little slower, but simplifies things by ensuring it behaves the same in development and in the built distributable.